packages/validator/src/api/interface/validators.ts

```diff
@@ -1,13 +1,5 @@
 import {BLSPubkey, CommitteeIndex, Epoch, Root, phase0, Slot, ValidatorIndex} from "@chainsafe/lodestar-types";
 
-export type BeaconCommitteeSubscription = {
-  validatorIndex: number;
-  committeeIndex: number;
-  committeesAtSlot: number;
-  slot: number;
-  isAggregator: boolean;
-};
-
```

Comment:
Type is also declared and used here https://github.com/ChainSafe/lodestar/blob/683af73def6c5150e86c75477be98cedefd11a15/packages/lodestar/src/api/impl/validator/interface.ts#L6